src/shipping/consignment-request-sender.ts

```diff
@@ -29,6 +30,10 @@ export default class ConsignmentRequestSender {
             ...SDK_VERSION_HEADERS,
         };
 
+        if (isMLIExperimentEnabled) {
+            DEFAULT_INCLUDES.push(CheckoutIncludes.AvailablePickupOptions);
```

Comment:
The API doesn't have `consignments.availablePickupOptions` as an optional include so you can remove this.